app/controllers/api/v1/votes_controller.rb

```diff
@@ -22,6 +22,14 @@ def destroy
     end
   end
 
+  def index
```

Comment:
If we want show then use `resource` singular in the routes. A singular resource doesn't have an index. Like the vote_counts.